tools/resourcedocsgen/cmd/metadata.go

```diff
@@ -0,0 +1,127 @@
+	"digitalocean":  pkg.PackageCategoryCloud,
+	"dnsimple":      pkg.PackageCategoryNetwork,
+	"docker":        pkg.PackageCategoryInfrastructure,
+	"eks":           pkg.PackageCategoryCloud,
+	"f5bigip":       pkg.PackageCategoryNetwork,
+	"fastly":        pkg.PackageCategoryCloud,
+	"gcp":           pkg.PackageCategoryCloud,
+	"google-native": pkg.PackageCategoryCloud,
+	"github":        pkg.PackageCategoryVCS,
+	"gitlab":        pkg.PackageCategoryVCS,
+	"hcloud":        pkg.PackageCategoryCloud,
+	"kafka":         pkg.PackageCategoryInfrastructure,
+	"keycloak":      pkg.PackageCategoryInfrastructure,
+	"kong":          pkg.PackageCategoryInfrastructure,
+	"kubernetes":    pkg.PackageCategoryCloud,
+	"linode":        pkg.PackageCategoryCloud,
+	"mailgun":       pkg.PackageCategoryInfrastructure,
+	"mongodbatlas":  pkg.PackageCategoryDatabase,
+	"mysql":         pkg.PackageCategoryDatabase,
+	"newrelic":      pkg.PackageCategoryMonitoring,
+	"ns1":           pkg.PackageCategoryNetwork,
+	"okta":          pkg.PackageCategoryInfrastructure,
+	"openstack":     pkg.PackageCategoryCloud,
+	"opsgenie":      pkg.PackageCategoryInfrastructure,
+	"pagerduty":     pkg.PackageCategoryInfrastructure,
+	"postgresql":    pkg.PackageCategoryDatabase,
+	"rabbitmq":      pkg.PackageCategoryInfrastructure,
+	"rancher2":      pkg.PackageCategoryInfrastructure,
+	"random":        pkg.PackageCategoryUtility,
+	"signalfx":      pkg.PackageCategoryMonitoring,
+	"spotinst":      pkg.PackageCategoryInfrastructure,
+	"tls":           pkg.PackageCategoryUtility,
+	"vault":         pkg.PackageCategoryInfrastructure,
+	"venafi":        pkg.PackageCategoryInfrastructure,
+	"vsphere":       pkg.PackageCategoryCloud,
+	"wavefront":     pkg.PackageCategoryMonitoring,
+	"equinix-metal": pkg.PackageCategoryCloud,
+	"splunk":        pkg.PackageCategoryInfrastructure,
+	"yandex":        pkg.PackageCategoryCloud,
+	"rke":           pkg.PackageCategoryInfrastructure,
+	"sumologic":     pkg.PackageCategoryMonitoring,
+}
+
+func packageMetadataCmd() *cobra.Command {
+	var metadataOutDir string
+	var featured bool
+
+	cmd := &cobra.Command{
+		Use:   "metadata <metadataOutDir> [featured]",
+		Short: "Generate package metadata only",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			status := pkg.PackageStatusGA
+			if strings.HasPrefix(version, "v0.") {
+				status = pkg.PackageStatusPublicPreview
+			}
+
+			category := pkg.PackageCategoryCloud
+			if c, ok := categoryLookup[mainSpec.Name]; ok {
+				category = c
+			}
+
+			pm := pkg.PackageMeta{
+				Name:          mainSpec.Name,
+				UpdatedOn:     time.Now().Unix(),
+				Publisher:     "Pulumi",
+				Title:         mainSpec.Name,
+				Description:   mainSpec.Description,
+				Category:      category,
+				PackageStatus: status,
+				Featured:      featured,
+				Native:        mainSpec.Attribution == "",
+				Version:       version,
+				Logo:          fmt.Sprintf("/logos/pkg/%s.svg", mainSpec.Name),
```

Comment:
Let's tweak this line to exclude the specifics about these file paths and extensions and just expect that the templates in the registry repo will use the package name to assemble the image references.